testsuite/pytests/test_labeled_synapses.py

```diff
@@ -56,7 +56,7 @@ def default_network(self, syn_model):
 
         self.urbanczik_synapses = ["urbanczik_synapse", "urbanczik_synapse_lbl", "urbanczik_synapse_hpc"]
 
-        self.eprop_synapses = ["eprop_synapse", "eprop_synapse_lbl", "eprop_synapse_hpc"]
+        self.eprop_synapses = ["eprop_synapse"]
```

Comment:
```suggestion
        self.eprop_synapses = ["eprop_synapse", "eprop_synapse_lbl", "eprop_synapse_hpc"]
```
The `_lbl` and `_hpc` variants of `eprop_synapse` are required for the test to pass.

Reply:
Sorry, my mistake.